app/views/layouts/_help_modal.erb

```diff
@@ -0,0 +1,18 @@
+<div class="modal modal-help" id="helpModal" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true">
+  <div class="modal-dialog modal-lg" style="z-index: 30; padding-top: 25vh;">
```

Comment:
is it necessary to have styles here? as I can see - you can use parent class name '.modal-help {.modal-dialog {...}}' and set those styles there.... how do you think?